src/FieldDataPluginFramework.Serialization.UnitTests/JsonConfigTests.cs

```diff
@@ -263,7 +309,7 @@ private FieldVisitInfo CreateFieldVisitInfo()
 
             visit.ControlConditions.Add(Fixture.Create<ControlCondition>());
 
-            foreach (var item in Fixture.CreateMany<DischargeActivity>())
+            foreach (var item in Fixture.CreateMany<DischargeActivity>(ChannelMeasurementTypeCount))
```

Comment:
This forces AutoFixture to create at least one instance of each channel type.

Reply:
I suppose another way of doing this more explicitly would be to do enumerate over the `ChannelMeasurementType` and create a discharge activity with one channel measurement created from the enum value, but this works as well. Also means multiple channels in the same activity are tested.